packages/klighd-core/src/options/options-panel.tsx

```diff
@@ -28,12 +28,20 @@ import { SidebarPanel } from "../sidebar";
 /** Sidebar panel that displays server provided KLighD options.  */
 @injectable()
 export class OptionsPanel extends SidebarPanel {
+    // sets this panel at the top position
+    // hirarchy is: first = -10; middle = 0; last = 10;
+    readonly position = -10;  // --> first position                                                            
     @inject(DISymbol.OptionsRegistry) private optionsRegistry: OptionsRegistry;
     @inject(DISymbol.OptionsRenderer) private optionsRenderer: OptionsRenderer;
-
+    
     @postConstruct()
     init(): void {
         this.optionsRegistry.onChange(() => this.update());
+        this.assignQuickActions();
+    }
+    update(): void {
```

Comment:
always have an empty line between methods (should usually also be enforced by prettier)